components/cart/CartItemActionsMobile/CartItemActionsMobile.tsx

```diff
@@ -1,12 +1,16 @@
 import React, { MouseEvent, useState } from 'react'
 
 import MoreVert from '@mui/icons-material/MoreVert'
-import { IconButton, Menu, MenuItem } from '@mui/material'
+import { IconButton, Menu, MenuItem, CardActions, Button } from '@mui/material'
 import { useTranslation } from 'next-i18next'
 
+import type { CrCartItem as CartItemType, Maybe } from '@/lib/gql/types'
+
 interface CartItemActionsMobileProps {
+  cartItem: any
```

Comment:
Remove any and provide a proper type. Check types.ts file for this.

Reply:
added proper type